packages/entities/entities-shared/src/components/entity-form-section/EntityFormSection.vue

```diff
@@ -75,6 +79,7 @@ defineProps({
 })
 
 const slots = useSlots()
+const legendID = uuidv4()
```

Comment:
```suggestion
const legendId = uuidv4()
```

nit: may be a personal preference, but I prefer every (non-first) word in camel-cased const name to start with a title latter and then continue in lower case (`ID -> Id`)